pkg/v3/hooks/prebuild/coordinate_block.go

```diff
@@ -0,0 +1,33 @@
+package prebuild
+
+import (
+	ocr2keepersv3 "github.com/smartcontractkit/ocr2keepers/pkg/v3"
+	"github.com/smartcontractkit/ocr2keepers/pkg/v3/instruction"
+)
+
+const (
+	ShouldCoordinateBlock instruction.Instruction = "should coordinate block"
+	DoCoordinateBlock     instruction.Instruction = "do coordinate block"
+)
+
+type coordinateBlockHook struct {
+	instructionStore instruction.InstructionStore
+}
+
+func NewCoordinateBlockHook(instructionStore instruction.InstructionStore) *coordinateBlockHook {
+	return &coordinateBlockHook{
+		instructionStore: instructionStore,
+	}
+}
+
+func (h *coordinateBlockHook) RunHook(outcome ocr2keepersv3.AutomationOutcome) error {
+	for _, instruction := range outcome.Instructions {
+		if instruction == ShouldCoordinateBlock {
+			h.instructionStore.Set(DoCoordinateBlock)
+			h.instructionStore.Delete(ShouldCoordinateBlock)
+			break
+		}
```

Comment:
I missed this one earlier, but we should also remove the do instruction when a coordinated block comes in the outcome. I don't think we have a key yet for the outcome metadata for coordinated block, but if that key exists, we should remove the do instruction and send the coordinated block to the coordinated block ticker. That ticker is in another PR, but you can just add a dependency for it in this run hook.

Reply:
Do you know what PR that ticker is in? The block ticker that was merged before was for block histories, but we need one for ocr2keepers.BlockKey, is that right?